acceptance/lib/puppet/acceptance/install_utils.rb

```diff
@@ -6,11 +6,12 @@ module Puppet
   module Acceptance
     module InstallUtils
       PLATFORM_PATTERNS = {
-        :redhat        => /fedora|el|centos/,
         :debian        => /debian|ubuntu/,
         :debian_ruby18 => /debian|ubuntu-lucid|ubuntu-precise/,
+        :redhat        => /fedora|el|centos/,
+        :sles          => /sles/,
```

Comment:
oh whoops.  i'd added that to some puppet topic branch i had to test
something against sles.  i must have been on that branch when i copied over
install_utils.rb

On Wed, Feb 25, 2015 at 2:13 PM, Kylo Ginsberg <notifications@github.com>
wrote:

> In acceptance/lib/puppet/acceptance/install_utils.rb
> <https://github.com/puppetlabs/facter/pull/856#discussion_r25384860>:
>
> >          :debian        => /debian|ubuntu/,
> >          :debian_ruby18 => /debian|ubuntu-lucid|ubuntu-precise/,
> > +        :redhat        => /fedora|el|centos/,
> > +        :sles          => /sles/,
>
> No objection at all but just curious, why you added sles?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/puppetlabs/facter/pull/856/files#r25384860>.
>



-- 
Eric Thompson
eric.thompson@puppetlabs.com <erict@puppetlabs.com>
Senior Software Quality Assurance Engineer